plugins/github-dev/commands/cr-wait.md

```
@@ -0,0 +1,108 @@
+
+Some PRs already have the status before the user calls `/cr-wait`. Probe once first:
+
+```bash
+gh api "repos/$OWNER/$REPO/commits/$SHA/status" \
+  --jq '.statuses[] | select(.context | test("CodeRabbit"; "i")) | {state, target_url, context, updated_at}'
+```
+
+If `state` is `success` or `failure`, emit the JSON line below and EXIT immediately:
+
+```json
+{"state":"success","sha":"<SHA>","pr":<PR_NUM>,"target_url":"<URL>","source":"probe"}
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)